### PR TITLE
Tweak: Add a tooltip for using globals [ED-9974]

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: assets/dev/js/editor/controls/popover-toggle.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export default class ControlPopoverStarterView extends ControlChooseView {
 		// Attach the current control as a toggle of its popover.
 		if ( $popover.length ) {
 			$popover[ 0 ].dataset.popoverToggle = `elementor-control-default-${ this.model.cid }`;
+			$popover[ 0 ].dataset.popoverCid = this.model.cid;
```
Comment (Nevoss) [inline assets/dev/js/editor/controls/popover-toggle.js:26]: Consider remove

File: assets/dev/js/editor/editor-base.js [hunk 2]
```diff
@@ -76,9 +77,13 @@ export default class EditorBase extends Marionette.Application {
 	}
 
 	backgroundClickListeners = {
+		tooltip: {
+			element: '#elementor-element--tooltip__dialog',
```
Comment (Nevoss) [inline assets/dev/js/editor/editor-base.js:81]: Consider changing to class

File: assets/dev/js/editor/modules.js [hunk 3]
```diff
@@ -13,6 +14,7 @@ elementorModules.editor = {
 	utils: {
 		Module: EditorModule,
 		Introduction,
+		Tooltip,
```
Comment (Nevoss) [inline assets/dev/js/editor/modules.js:17]: Consider using `Introduction`

File: assets/dev/js/editor/utils/tooltip.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+		return {
+			dialogType: 'tooltip',
+			dialogOptions: {
+				id: 'elementor-element--tooltip__dialog',
+				effects: {
+					show: 'show',
+					hide: 'hide',
+				},
+				hide: {
+					// TODO: Ask Roi if he wants to keep the tooltip on change of the popovers.
```
Comment (Nevoss) [inline assets/dev/js/editor/utils/tooltip.js:16]: Remove

File: assets/dev/js/editor/tooltips/tooltips/globals-introduction.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+						__( 'Save time by applying Global Colors to change the style of multiple elements at once. Click %s to see what Global Colors you already have.', 'elementor' ),
+						"<i class='eicon-globe'></i>",
+					),
+				);
+				this.showTooltip( e.detail.el );
+				this.setViewed();
+			}
+		} );
+
+		window.addEventListener( 'elementor/popover/show', ( e ) => {
```
Comment (Nevoss) [inline assets/dev/js/editor/tooltips/tooltips/globals-introduction.js:33]: Split into 2 classes

File: tests/playwright/sanity/assets/dev/js/editor/tooltips/globals-introduction.test.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+			elementor.config.user.introduction.globals_introduction_tooltip = false;
+
+			// Register the tooltips again after we set the introduction to false.
+			elementor.tooltips.registerTooltips();
+		} );
+
+		// Act.
+		await editor.addWidget( 'heading' );
+		await editor.activatePanelTab( 'style' );
+		await page.click( '.pickr' );
```
Comment (Nevoss) [inline tests/playwright/sanity/assets/dev/js/editor/tooltips/globals-introduction.test.js:20]: Be more specific

File: tests/playwright/sanity/assets/dev/js/editor/tooltips/globals-introduction.test.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+		let tooltipDialogCounter = await page.locator( '.dialog-tooltip-widget' ).count();
```
Comment (Nevoss) [inline tests/playwright/sanity/assets/dev/js/editor/tooltips/globals-introduction.test.js:21]:
```suggestion
		let tooltipDialogCounter = await page.locator( 'text="Check out Global Fonts"' ).count();
```

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/assets/dev/js/editor/tooltips/globals-introduction.test.js (hunk 7 above)
Comment (yotamselementor) [reply tests/playwright/sanity/assets/dev/js/editor/tooltips/globals-introduction.test.js:21]: Or, by using the newish API: https://playwright.dev/docs/api/class-page#page-get-by-text
```suggestion
		let tooltipDialogCounter = await page.getByText( 'Check out Global Fonts' ).count();
```

File: assets/dev/js/editor/tooltips/manager.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+		const tooltips = [
+			new GlobalColorIntroduction( 'globals_introduction' ),
+			new GlobalFontIntroduction( 'globals_introduction' ),
+		];
+
+		tooltips.forEach( ( tooltip ) => {
+			if ( ! elementor.config.user.introduction[ tooltip.introductionKey ] ) {
+				tooltip.initTooltip();
+				tooltip.bindEvent();
+			}
```
Comment (Nevoss) [inline assets/dev/js/editor/tooltips/manager.js:39]: Nice 💪🏽

**Review by Nevoss — DISMISSED** [review]

File: assets/dev/js/editor/introduction-tooltips/manager.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
+import GlobalColorIntroduction from './tooltips/global-color-introduction';
+import GlobalFontIntroduction from './tooltips/global-font-introduction';
+
+export default class TooltipsManager {
```
Comment (Nevoss) [inline assets/dev/js/editor/introduction-tooltips/manager.js:4]:
```suggestion
export default class IntroductionTooltipManager {
```

File: assets/dev/js/editor/introduction-tooltips/tooltips/global-font-introduction.js [hunk 10]
```diff
@@ -0,0 +1,61 @@
+export default class GlobalColorIntroduction {
+	introductionKey;
+
+	constructor( introductionKey ) {
+		this.introductionKey = introductionKey;
+	}
+
+	getIntroductionViewed() {
```
Comment (Nevoss) [inline assets/dev/js/editor/introduction-tooltips/tooltips/global-font-introduction.js:8]: Where is it used?

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/introduction-tooltips/manager.js (hunk 9 above)
Comment (ManorHazazE) [reply assets/dev/js/editor/introduction-tooltips/manager.js:4]: Done

File: assets/dev/js/editor/introduction-tooltips/tooltips/global-font-introduction.js (hunk 10 above)
Comment (ManorHazazE) [reply assets/dev/js/editor/introduction-tooltips/tooltips/global-font-introduction.js:8]: Removed

**Review by Nevoss — APPROVED** [review]

**Pull request merged**